crates/sdk/src/client_cache.rs

```diff
@@ -238,7 +239,11 @@ impl<T: TableType> TableCache<T> {
                         );
                     }
                     Ok(row) => {
-                        log::info!("Initializing table {:?}: got new row {:?}", T::TABLE_NAME, row);
+                        if log_enabled!(log::Level::Trace) {
+                            log::trace!("Initializing table {:?}: got new row {:?}", T::TABLE_NAME, row);
+                        } else {
+                            log::debug!("Initializing table {:?} with new row", T::TABLE_NAME);
+                        }
```

Comment:
Well, your call @gefjon. In my debugging, it was _usually_ useful to observe this things, _sometimes_ horrible to see the row (because it was too large), and _occasionally_ useful to see the row, in particular to convince myself that a proper update was executed (i.e. delete followed by insert).

Reply:
Alright, if you've found the debug-level logs useful, I'm happy to have them. And given we're keeping them, I don't particularly want a macro obscuring what's going on here.